.eslintrc.js

```diff
@@ -472,6 +472,42 @@ module.exports = {
         ],
       },
     },
+    {
+      files: ['./packages/desktop-client/**/*'],
+      excludedFiles: ['./packages/desktop-client/src/redux/index.ts'],
+      rules: {
+        'no-restricted-imports': [
+          'warn',
+          {
+            patterns: [
+              {
+                group: ['react-redux'],
+                importNames: ['useDispatch'],
+                message: 'Please use Actual’s useDispatch() hook instead.',
```

Comment:
nit: maybe we should specify where to find Actual's hook? e.g. 
```suggestion
                message: 'Please import useDispatch() from `src/redux` instead.',
```